src/description/eth_accounts.md

```diff
@@ -0,0 +1 @@
+* If the cleint owns any ethereum addresses it **MUST** return them, otherwise it **MUST** return an empty array.
```

Comment:
We should be more clear about "owns" here and instead say something like "has private keys for".

The client can return whatever subset of addresses it wants.  The user may tell the client "only tell dapps about accounts 1 and 3 but not 2 and 4" and the client must respect that.  Similarly, the client may prompt the user before returning any accounts and ask the user which (if any) they want to return.

Even then, it is quite reasonable for a client or wallet to return an address it doesn't have signing keys for such as if you have a view-only wallet you are using.

TL;DR: I recommend removing this line (and thus file I guess) entirely.